packages/plugins/experimental/plugin-calls/src/components/Call/Call.tsx

```diff
@@ -56,32 +62,37 @@ export const Call: FC<ThemedClassName> = ({ classNames }) => {
 
   useBroadcastStatus({ userMedia, peer, updateUserState, identity, pushedTracks, ai, speaking: isSpeaking });
 
+  const transcriptionProps = useHandleTranscription();
```

Comment:
the number 5 is hardcoded above; if that's indented move it to a const so it's clearer what that it.